packages/react/src/theme/components/timepicker.ts

```diff
@@ -0,0 +1,43 @@
+import { defineStyle, defineStyleConfig } from "@chakra-ui/react";
+
+const baseStyle = defineStyle({});
+
+const variantOutline = defineStyle((props) => {
```

Comment:
Jeg skjønner ikke helt hvorfor css-en her ligger under variantOutline, den gjelder jo (og ser forsåvidt også ut til å fungerer for) alle variantene. Det som er spesifikt for outline vs de andre variantene ligger jo ikke her.